CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ project(chimera-sdk LANGUAGES C ASM)
 set(ABI ilp32)
 set(ISA_CLUSTER_SNITCH rv32im)
 set(ISA_HOST rv32imc)
+set(DISASSEMBLE_LIBRARIES CACHE BOOL OFF)
```

Comment:
I'd argue it almost always makes sense to just provide the objdump for all binaries. If you don't need it you won't feel the overhead, if you need it, you don't want to go digging in the make flow.

Reply:
Disassembling of the libraries is enabled per default. This flag only triggers to also disassemble the static libraries.